tf2_py/CMakeLists.txt

```diff
@@ -1,26 +1,47 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_py)
 
-if(NOT WIN32)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra -Wpedantic")
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra)
```

Comment:
Do we need these flags on for some reason? I'd prefer these only get introduced if we're compiling with `-DCMAKE_BUILD_TYPE=Debug`.